scripts/build-shotcut.sh

```diff
@@ -1200,6 +1220,13 @@ EOF
   cmd ninja install
 }
 
+function install_whisper {
+  cmd ninja -C build install
+  cmd install -p -c build/bin/main $FINAL_INSTALL_DIR/bin/whisper_main
+  cmd mkdir -p $FINAL_INSTALL_DIR/share/shotcut/whisper_models
+  cmd install -p -c models/ggml-base-q5_1.bin $FINAL_INSTALL_DIR/share/shotcut/whisper_models
```

Comment:
Another point of discussion: I chose to install the openai model in share/shotcut. Is that the right place? I think so because whisper.cpp does not have a "share" install target. Nor does Whisper.cpp provide any recommendation for where to install the models.

Reply:
This is OK.